src/UXClient/Components/EventsPlot/EventsPlot.ts

```diff
@@ -1,7 +1,7 @@
 import * as d3 from 'd3';
 import './EventsPlot.scss';
 import { Plot } from '../../Interfaces/Plot';
-import { Utils, NONNUMERICTOPMARGIN, EventElementTypes, DataTypes, LINECHARTTOPPADDING } from '../../Utils';
+import Utils, { NONNUMERICTOPMARGIN, EventElementTypes, DataTypes, LINECHARTTOPPADDING } from '../../Utils';
 
 const TOPMARGIN = 4;
 
```

Comment:
You might have mentioned this already, but why are some of the components changing to default exports, but not other ones like this EventsPlot?

Reply:
@darsney, what do you think about this.  Should we transition all Components to default exports in this PR?

Reply:
I'm thinking that we could make a separate smaller PR if we want to make that change.

This just touches the public components that we're building individual imports for.

It's already pretty big, so I think that may make it harder to finish this review.

Reply:
Leaving for another PR.